randovania/game_description/requirements.py

```diff
@@ -177,10 +179,10 @@ def damage(self, current_resources: ResourceCollection, database: ResourceDataba
             return MAX_DAMAGE
 
     def satisfied(self, current_resources: ResourceCollection, current_energy: int, database: ResourceDatabase) -> bool:
-        return any(
-            item.satisfied(current_resources, current_energy, database)
-            for item in self.items
-        )
+        for item in self.items:
+            if item.satisfied(current_resources, current_energy, database):
+                return True
+        return False
```

Comment:
I'm surprised this is faster tbh

Reply:
I didn't test and don't expect it to be, but it's also simpler and in one of the hottest functions.